### ISSUE TITLE
Communicate with OMERO headless in a conda environment

### ISSUE THREAD

**Issue by lldelisle** [GReD-Clermont/simple-omero-client#33]
Dear developer,
I wrote a workflow to use in galaxy (https://galaxyproject.org/) to be able to get images from omero, analyze with Fiji, send back to omero both results and ROI. If you don't know galaxy, or how it works. Galaxy is a web-based graphical interface which will run command lines in the backend thanks to 'tool wrappers'. Dependencies are solved by conda and more recently by singularity.
My existing workflow was using a python script to get images from omero (using omero-py), then a fiji macro (using fiji in headless mode), then again a python script to send back data (using omero-py).
@romainGuiet told me about your plugin which allow to interact with omero directly from fiji using (for example) a groovy macro.
I was wondering if you would be interested in having your plugin available in conda?
If yes, could we write a conda recipe for your plugin? (if it is a just a jar, writting a simple recipe like https://github.com/bioconda/bioconda-recipes/tree/master/recipes/fiji-morpholibj should work but it looks like you have dependencies, then could you help me to identify them and contact the developers to see if they also agree to put them on conda).
Thank you very much

**Comment by ppouchin** [comment]
Dear Lucille, 

I'd be happy to add it to Conda! Moreover, we do have an uber-jar, but I'm still unsure about the best way to handle dependencies. I fear that deploying this with OMERO included could conflict with other plugins. 
This is something I'd like to sort out as putting this on a Fiji update site would be important as well. 

I'll keep this open, and come back here when I have a better idea on how to handle this.

**Comment by lldelisle** [comment]
Dear @ppouchin ,
Just to tell you that I am still interested in your plugin in conda, if you have time to think about the dependencies...
Thanks

**Comment by ppouchin** [comment]
I'm very very late on all this, but I think it will be quick once #41 is solved (which should be soon: the library will be uploaded to the OMERO update site after I rework it a bit)

**Comment by lldelisle** [comment]
This would be great. However, I am not sure, #41 will solve directly the conda issue we may need to work a bit more to make it as a conda package. Let see.

**Comment by ppouchin** [comment]
Yes, in practice the library works when the OMERO.insight plugin is installed in Fiji. That's what my local "update site" for my lab provides. People from the OME team have warned me that it may not be optimal to do so though (in the report on our paper: https://f1000research.com/articles/11-392/v1#referee-response-129917). 

Also, you were right that an update site does not solve this issue since update sites, well... always ***update***, and thus cannot be used. And the update site I want to join for the dependencies ("OMERO-5.5-5.6") probably does not provide a conda package either.

**Comment by lldelisle** [comment]
Youhou:
https://anaconda.org/bioconda/fiji-simple_omero_client
Thank you
